src/odemis/acq/stream/_projection.py

```diff
@@ -936,14 +936,27 @@ def getRawValue(self, pixel_pos):
         raw = self.stream.raw
         if not raw:
             raise LookupError("Failed to find raw data for %s stream" % (self.stream,))
+
```

Comment:
Can you update the docstring, because now it does not return an array if it has a maximum intensity projection and a z-dimension

```python

        Returns: the raw "value" of the position. In case the raw data has more than 2 dimensions, it returns an array.
        Raise LookupError if raw data not found
```